resume_pj/index.html

```diff
@@ -0,0 +1,148 @@
+               
+                    <img src="./img/photo-profile.jpeg" class="rounded-circle col-5" alt="profile pic">
+                    <h2>Maya Nelson</h2>
+                    <hr>
+                   
+                        <div class="row justify-content-around">
+                            
+                                <button type="button" class="col-5 btn btn-outline-primary">RESUME</button>
+                            
+                            
+                                <button type="button" class="col-5 btn btn-outline-primary">PROJECTS</button>
+                        </div>
+                        </div>
+                    <div class="row social-class justify-content-center">
+                        <div class="col-1">
+                            <a href="">
+                            <i class="fa fa-instagram"></i>
+                            </a>
+                        </div>
+                        <div class="col-1">
+                            <a href="">
+                                <i class="fa fa-facebook"></i>
+                            </a>
+                        </div>
+                        <div class="col-1">
+                            <a href="">
+                                <i class="fa fa-twitter" style="color: rgb(32, 32, 32);"></i> 
+                            </a>
+                        </div>
+                        <div class="col-1">
+                            <a href="">
+                                <i class="fa fa-linkedin-square"></i>
+                            </a>
+                        </div>
+                    </div>
+                    
+                    </div>
+                </div>
+            </div>
+                 <div class="border border-danger col-md-7 col-12 hello-section">
+                    <h1>Hello</h1>
+                    <h3>Here's who I am & what I do</h3>
+                    <p>
+                        I'm a paragraph. Click here to add your own text and edit me. It’s easy. Just click “Edit Text” or double click me to add your own content and make changes to the font.
+
+ 
+
+I’m a great place for you to tell a story and let your users know a little more about you.
+                    </p>
+               
+           
+        </div>
+
+
+    </main>
+    <footer class="container-fluid">
+        <div class="row">
+            
+               
+            <div class="col-6">
+                <h3>Call</h3>
+                <h4>123-456-7890</h4>
+            </div>
+                
+            
+            <div class="col-6">
+               <h3>Write</h3>
+               <h4>123-456-7890</h4>
+                
+            </div>
+            <div class="col-12">
+                <h3>Follow</h3>
+                <div class="row social-class justify-content-center">
+                    <div class="col-1">
+                        <a href="">
```

Comment:
Correct use of Bootstrap here re-using `row` when nesting columns inside columns. I'm not sure this was necessary though. You can just take this whole section from line 116 to 140 and take it outside of the `row` from line 102, and put in inside its own `row`, like this:

```
<footer class="container-fluid">
     <div class="row">
            <div class="col-6">
                <h3>Call</h3>
                <h4>123-456-7890</h4>
            </div>
            <div class="col-6">
               <h3>Write</h3>
               <h4>123-456-7890</h4>
            </div>
      </div> // close the first row
      <div class="row> // open a new one
            <div class="col-12">
                <h3>Follow</h3>
            </div> // give the 'follow' heading its own 12 columns if you always want it to occupy the full width
            <div class="col-12 social-class justify-content-center"> // no need for a new row here
                    <div class="icon-wrapper"> // I don't think you need to use col-1 here, just give them a custom class like 'icon-wrapper' with fixed width (e.g. 50px)
                        <a href="">
                            <i class="fa fa-instagram"></i>
                        </a>
                    </div>
                    <div class="icon-wrapper">
                        <a href="">
                            <i class="fa fa-facebook"></i>
                        </a>
                    </div>
                    (etc for other icons)
```